tests/integration/api/controllers/iac/cloudformation/test_otm_controller_iac_cloudformation.py

```diff
@@ -65,10 +67,10 @@ def test_create_otm_ok(self):
         assert '"trustZones": ' in response.text
         assert '"components": ' in response.text
 
-    @mark.parametrize('project_id,project_name,cft_filename,cft_mimetype,mapping_filename,error_type',
-                      [uc_a, uc_b, uc_c, uc_d, uc_e, uc_f, uc_h, uc_i, uc_j, uc_k])
+    @mark.parametrize('project_id,project_name,cft_filename,cft_mimetype,mapping_filename,default_mapping_file,error_type',
+                      [uc_m])
```

Comment:
The dataset: 
[uc_a, uc_b, uc_c, uc_d, uc_e, uc_f, uc_h, uc_i, uc_j, uc_k, uc_l] are missing, is it correct?